iotdb-core/pom.xml

```diff
@@ -38,4 +38,12 @@
         <module>node-commons</module>
         <module>relational-grammar</module>
     </modules>
+    <profiles>
+        <profile>
+            <id>with-ainode</id>
+            <modules>
+                <module>ainode</module>
+            </modules>
+        </profile>
+    </profiles>
```

Comment:
Good job!
Is there any plan to add a profile in distribution module? When `with-ainode` is activated, the package can be packed in distribution.

Reply:
That makes sense! I will try that.

Reply:
> Good job! Is there any plan to add a profile in distribution module? When `with-ainode` is activated, the package can be packed in distribution.

Hi! I've found some aspects of this feature that are worth discussing, so I plan to develop it as a new feature after this PR is merged, which will facilitate our discussion.